src/tests/possumwood/json.cpp

```diff
@@ -31,10 +32,22 @@ dependency_graph::NodeBase& findNode(const std::string& name) {
 	return findNode(possumwood::AppCore::instance().graph(), name);
 }
 
+json readJson(possumwood::IFilesystem& filesystem, const std::string& filename) {
+	json result;
+
+	auto stream = filesystem.read(possumwood::Filepath::fromString(filename));
```

Comment:
Codacy found an issue: [Check buffer boundaries if used in a loop including recursive loops (CWE-120, CWE-20).  ](https://app.codacy.com/manual/martin.prazak/possumwood/pullRequest?prid=6093062)